src/shim/shim_logger.c

```diff
@@ -17,10 +17,11 @@ typedef struct _ShimLogger {
     FILE* file;
 } ShimLogger;
 
-static uint64_t _simulation_nanos;
+static uint64_t _simulation_nanos = 0;
 void shimlogger_set_simulation_nanos(uint64_t simulation_nanos) {
     _simulation_nanos = simulation_nanos;
 }
+uint64_t shimlogger_get_simulation_nanos() { return _simulation_nanos; }
```

Comment:
Is there a reason why sometimes the `shimlogger_get_simulation_nanos()` is used, and sometimes the variable is accessed directly?

Reply:
Within the shim logger, `_simulation_nanoseconds` can be used directly because it is local to the shim logger. I made an access function so that I can retrieve it externally from `preload_syscall.c`.